MultisigIntegrationTests/CoreData/NetworkTests.swift

```diff
@@ -68,48 +85,95 @@ class NetworkTests: CoreDataTestCase {
         XCTAssertThrowsError(try makeNetwork(id: 1))
     }
 
-    // update(from networkInfo)
-        // when different chain id then throws
-        // when set the params, then updates with appropriate params
-        // when no theme exists, creates it
```

Comment:
I'm missing these things tested: can it be that network created without theme or native currency attached to it?